auditlog/models/log.py

```diff
@@ -106,3 +91,75 @@ def write(self, vals):
                 {"field_name": field.name, "field_description": field.field_description}
             )
         return super().write(vals)
+
+
+class AuditlogLogLineView(models.Model):
+    _name = "auditlog.log.line.view"
```

Comment:
If you both use name and inherits, it could avoir to redefine all the fields, that all comes from log and log line models, dont you think ?